cmd/spire-server/cli/entry/delete.go

```diff
@@ -73,12 +118,30 @@ func (c *deleteCommand) prettyPrintDelete(env *commoncli.Env, results ...interfa
 		return cliprinter.ErrInternalCustomPrettyFunc
 	}
 
-	sts := deleteResp.Results[0].Status
-	switch sts.Code {
-	case int32(codes.OK):
-		env.Printf("Deleted entry with ID: %s\n", c.entryID)
-		return nil
-	default:
-		return fmt.Errorf("failed to delete entry: %s", sts.Message)
+	var failed, succeeded []*entryv1.BatchDeleteEntryResponse_Result
+	for _, result := range deleteResp.Results {
+		switch result.Status.Code {
+		case int32(codes.OK):
+			succeeded = append(succeeded, result)
+		default:
+			failed = append(failed, result)
+		}
 	}
+
+	for _, result := range succeeded {
+		env.Printf("Deleted entry with ID: %s\n", result.Id)
+	}
+
+	for _, result := range failed {
+		env.ErrPrintf("Failed to delete entry with ID %s (code: %s, msg: %q)\n",
```

Comment:
We should probably print out a newline or two between the success and error lines